internal/controller/helmchart_controller.go

```diff
@@ -1329,8 +1331,8 @@ func (r *HelmChartReconciler) makeVerifiers(ctx context.Context, obj *helmv1.Hel
 				Name:      secretRef.Name,
 			}
 
-			var pubSecret corev1.Secret
-			if err := r.Get(ctx, certSecretName, &pubSecret); err != nil {
+			pubSecret, err := r.retrieveSecret(ctx, obj.Namespace, secretRef.Name)
```

Comment:
Refactored `retrieveSecret()` so that `certSecretName` is returned. `certSecretName` is just used for logging. [160e237](https://github.com/fluxcd/source-controller/pull/1075/commits/160e23757b0c13e0ff3776599ae297ba39dd1e95)